articles/quickstart/webapp/nodejs/01-login.md

```diff
@@ -12,365 +12,103 @@ topics:
 contentType: tutorial
 useCase: quickstart
 github:
-    path: 01-Login
+  path: 01-Login
 ---
-<!-- markdownlint-disable MD002 -->
+<!-- markdownlint-disable MD002 MD041 -->
 
-<%= include('../_includes/_getting_started', { library: 'Node.js', callback: 'http://localhost:3000/callback' }) %>
+## Configure Auth0
 
-<%= include('../../../_includes/_logout_url', { returnTo: 'http://localhost:3000' }) %>
+You will need to register your application with Auth0 in order to start authenticating users. Go to the [Applications](${manage_url}/#/applications) screen in the Auth0 dashboard, create a new **Regular Web Application**, and follow the steps below.
 
-## Configure Node.js to use Auth0
+### 1. Configure Callback URL
```

Comment:
Sub-heads [should use sentence case](https://github.com/auth0/docs/blob/master/STYLEGUIDE.md#formatting):

```suggestion
### 1. Configure callback URL
```